src/control/server/external.go

```diff
@@ -123,7 +123,7 @@ func (e *ext) createEmpty(path string, size int64) (err error) {
 		if ok && (e == syscall.ENOSYS || e == syscall.EOPNOTSUPP) {
 			log.Debugf(
 				"Warning: Fallocate not supported, attempting Truncate: ", e)
-			err = file.Truncate(size)
+			_ = file.Truncate(size)
```

Comment:
Likewise here.

Reply:
as above, this error is not handled so we should fix in another patch and keep behaviour the same in this one.